docs/api.json

```diff
@@ -15,7 +15,7 @@
 				"isExported": true,
```

Comment:
Let's not include the api update, or do a manual search and replace of `\\` with `/`.